api/api_public_blockchain.go

```diff
@@ -359,7 +370,11 @@ func (s *PublicBlockChainAPI) EstimateComputationCost(ctx context.Context, args
 
 // EstimateGas returns an estimate of the amount of gas needed to execute the given transaction against the latest block.
 func (s *PublicBlockChainAPI) EstimateGas(ctx context.Context, args CallArgs) (hexutil.Uint64, error) {
-	return s.DoEstimateGas(ctx, s.b, args, s.b.RPCGasCap())
+	gasCap := uint64(0)
+	if rpcGasCap := s.b.RPCGasCap(); rpcGasCap != nil {
+		gasCap = rpcGasCap.Uint64()
+	}
+	return s.DoEstimateGas(ctx, s.b, args, big.NewInt(int64(gasCap)))
 }
```

Comment:
Are 350 ~ 353 lines same as this? If so, I think upper case better.  What do you think?